<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogImportDetails</class>
 <widget class="QDialog" name="DialogImportDetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBrowse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelSum">
       <property name="text">
        <string>Sum:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelAmountSum">
       <property name="text">
        <string>0 pcs.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPriceSum">
       <property name="text">
        <string>0 Ft</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButtonImport">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
